cantusdb/turtle/csv2rdf_single_subject.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import validators
+import sys
+import json
+import os
+from rdflib import Graph, URIRef, Literal
+from rdflib.namespace import RDF
+from typing import Optional
+
+dirname = os.path.dirname(__file__)
+mappername = os.path.join(dirname, 'relations_mapping.json')
+destname = os.path.join(dirname, 'out_rdf.ttl')
```

Comment:
```suggestion
DIRNAME = "."
mapping_filename = os.path.join(DIRNAME, 'relations_mapping.json')
dest_filename = os.path.join(DIRNAME, 'out_rdf.ttl')
```

You kind of re-engineered a more complicated relative file reference? I'm still not totally sure what your intention is. Do you want a user to be able to configure the file names and paths? 

This suggestion I guess shows a default to these file names in the current directory but then could be modified to not the current directory?

Reply:
I think if you assign "." to DIRNAME, it directs to the current position where you are running the script. Only __file__ is where the file is located.

Reply:
True. So the desired behaviour here is that you can run this script from anywhere but that those files must be/will be in the directory where the script lives?

Reply:
Yes, since this script applies to the CSV files from other databases.